mappings/net/minecraft/entity/mob/hoglin/HoglinBrain.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4835 net/minecraft/entity/mob/HoglinBrain
+CLASS net/minecraft/class_4835 net/minecraft/entity/mob/hoglin/HoglinBrain
```

Comment:
Isn't there a package like `entity.ai.brain` or so for classes like these?

Reply:
it's where hoglin brain handling logic is, must be in same package as hoglin